ansible/roles/software/wordpress/datagov-sudo-2-init/templates/datagov.nginx.conf

```diff
@@ -7,6 +7,7 @@ upstream php {
 server {
     listen 80 default_server;
     server_name _;
+    if ( $request_uri ~* "%0A|%0D" ) { return 403; }
```

Comment:
👍 should this also be applied to the https config?

Reply:
Yes, that probably makes sense though.